.circleci/config.yml

```diff
@@ -41,8 +41,11 @@ jobs:
       - run: PYTHON_VERSION=3.6 GIT_VERSION=1.8.0  bash ci/tox/ci-run.sh  # Earliest versions of python/git supported by git-machete
       - run: PYTHON_VERSION=3.7 GIT_VERSION=2.7.6  bash ci/tox/ci-run.sh
```

Comment:
I'd add `name:` to specify a human-friendly name of the stage (like `Run tests against Python 3.x and git x.y.z (with coverage)`)